tests/ndarrays/test_ndarrays.c

```diff
@@ -802,7 +802,7 @@ int32_t test_array_zeros_cdouble(void)
     double complex value;
     double complex c_value;
 
-    x = array_create(2, m_1_shape, nd_cdouble, false);
+    x = array_create(2, m_1_shape, nd_cdouble, false, order_c);
```

Comment:
Why not make one (or more) of these `order_f` ?

Reply:
Okey, it shall be done, I am having trouble understanding the purpose of this file, is it used to test the ndarray C library?

Reply:
Yes exactly